data/sea/10-mempool.h

```diff
@@ -66,66 +18,42 @@ static void imempool_add_block (imempool_t *pool)
         return ptr;                                                                                    \
     }
 
-static void * NOINLINE imempool_alloc_block (imempool_t *pool, size_t num_bytes)
+static void * NOINLINE imempool_alloc_block (anemone_mempool_t *pool, size_t num_bytes)
```

Comment:
the alloc for each entity check. I was considering checking the alloc'd amount somewhere in here, and return an error. But I think the idea discussed today (checking before/after compute in psv and zebra) is better anyway. So these functions can go